oof/src/lib.rs

```diff
@@ -220,6 +220,25 @@ mod tests {
         args.map(OsString::from).collect()
     }
 
+    fn setup_args_scenario(arg_str: &str) -> Result<(Vec<OsString>, Flags), OofError> {
+        let flags_info = [
+            ArgFlag::long("output_file").short('o'),
+            Flag::long("verbose").short('v'),
+            Flag::long("help").short('h'),
+        ];
+
+        let args = gen_args(arg_str);
+        filter_flags(args, &flags_info)
```

Comment:
compile error here, trying to return a value referencing local variable flags_info

Reply:
Sorry for the late response, I'll take a look now.

Reply:
Ok please update this PR, i'm interested in seeing how to tackle this scenario

Reply:
I made a commited into your branch, you can see the changes by clicking in `Files changed`, now that problem won't occur anymore.